src/utils/vanillaApi.ts

```diff
@@ -110,6 +110,43 @@ export async function getPopularDiscussions(): Promise<FormattedDiscussion[]> {
   }
 }
 
+export async function getAllEvents(): Promise<FormattedEvent[]> {
```

Comment:
what about the other methods? do we still use them? If not i would propose to remove them because the more code we have the more we need to maintain over time and in a year from now we dont know much about the other code anymore.